config/initializers/devise.rb

```diff
@@ -296,4 +296,6 @@
   # When set to false, does not sign a user in automatically after their password is
   # changed. Defaults to true, so a user is signed in automatically after changing a password.
   # config.sign_in_after_change_password = true
+
+  config.omniauth :facebook, '2426000817621711', '359ff024de4101ebe1c51ec0ce80a228'
```

Comment:
[OPTIONAL]

`FACEBOOK_APP_ID` and `FACEBOOK_APP_SECRET` should be not be committed.

Consider investigating the two gems:
1. [`dotenv`](https://github.com/bkeepers/dotenv)
2. [`figaro`](https://github.com/laserlemon/figaro)

Also, let's check [this link on configuring environment variables on Heroku](https://devcenter.heroku.com/articles/config-vars) for the next milestone.